draft-ietf-tls-dtls13.md

```diff
@@ -437,7 +439,8 @@ record consumes the entire rest of the datagram in the lower
 level transport. In this case it is not possible to have multiple
 DTLSCiphertext format records without length fields in the same datagram.
 Omitting the length field MUST only be used for the last record in a
-datagram.
+datagram. Implementations MAY mix records with and without length
+fields on the same connection.
```

Comment:
(I has assumed that "omitting the length only works for the last record in a datagram" implies that you can have a record with length and a record without length in the same datagram, and thus on the same connection, but don't mind saying it explicitly.)